README.md

```diff
@@ -6,39 +6,43 @@ TonY enables running either single node or distributed
 training as a Hadoop application. This native connector, together with other TonY features, aims to run
 machine learning jobs reliably and flexibly.
 
+## Compatibility Notes
+
+It is recommended to run TonY with [Hadoop 3.1.1](https://hadoop.apache.org/old/releases.html#8+Aug+2018%3A+Release+3.1.1+available) and above. TonY itself is compatible with Hadoop 2.7 and above. However, if you want to use TonY with docker, you need [Hadoop 2.9.1](https://hadoop.apache.org/docs/current/hadoop-yarn/hadoop-yarn-site/DockerContainers.html) or higher and if you need GPU isolation from TonY, you need [Hadoop 3.1.0](https://hortonworks.com/blog/gpus-support-in-apache-hadoop-3-1-yarn-hdp-3/) or higher.
```

Comment:
I think TonY requires 2.9+ since TonYApplicationMaster uses some YARN APIs only added in 2.8 or 2.9. Let's capitalize "Docker" (same elsewhere in this file (at least 7 other occurrences))

Reply:
Oh you're right. We did a round of clean up deprecated APIs.